### DISCUSSION TITLE
Could you please give me the full log when error happened?

### DISCUSSION THREAD

**Discussion by panelding** [PaddlePaddle/PaddleSpatial#243] [BaiduKDDCup2022]
Could you please give me the full log when error happened?

team name: AIStudio2391045Team
submit time: 2022-07-07 15:38
error I got: Err: 87.31% turbines with zero predicted values (in 0th prediction)!! 

I can run the evaluation.py you provided in my local environment.  
![image](https://user-images.githubusercontent.com/31956407/177727322-00547085-d6a0-471c-a14d-36e5e024bded.png)

please help 
thanks a lot

**Comment by shaido987** [comment]
I think I know the answer to your problem. The evaluation code was updated recently to change `min_non_zero_ratio` from 0.5 to 0.9. This is the relevant part of the code: https://github.com/PaddlePaddle/PaddleSpatial/blob/main/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/paddlepaddle/metrics.py#L203

So the problem is that you have too many zeros in the prediction (87.31% where the maximum allowed number is 10%).

(note that I'm just a participant and not one of the organizers so I could be wrong here)

**Reply by panelding** [reply]
I got , thanks very much!
